undertow/src/main/java/org/wildfly/extension/undertow/UndertowTransformers.java

```diff
@@ -100,6 +102,7 @@ private static void registerTransformers_EAP_7_1_0(ResourceTransformationDescrip
                 .addChildResource(UndertowExtension.PATH_APPLICATION_SECURITY_DOMAIN)
                 .getAttributeBuilder()
                     .addRejectCheck(RejectAttributeChecker.DEFINED, SECURITY_DOMAIN)
+                    .setDiscard(DiscardAttributeChecker.ALWAYS, ENABLE_JASPI, INTEGRATED_JASPI) // These are completely meaningless on the older host with no alternative required.
```

Comment:
@darranl Should we not reject if they are true? Otherwise people expect that jaspi will work on the older host, while their apps are unsecured

Reply:
I can reject if that is what we prefer, but this resource has meaning on it's own without that attribute.

Reply:
Actually true is the default value for this resource, if if anyone tries to set to true it should get swallowed up in being the default value.

Reply:
@bstansberry The part that complicates this was JASPI was not supported with WildFly Elytron in the prior versions at all so that is why 'false' can be safely discarded.

The default value of this attribute is 'true' which is what we want as there is a subsequent step required to really activate it which is also not satisified by default.

So it sounds like the only rejection I have is if defined and set to true.